### PR TITLE
Fix tests failing

### PR DESCRIPTION
**Pull request by andreycizov**
Latest pytest changed their API, so fixing it.

### CODE REVIEW

**Comment by andreycizov** [comment]
It makes sense to merge #185 first.

File: test/test_capability_context.py [hunk 1]
```diff
@@ -4,10 +4,13 @@
 
 this_dir = os.path.dirname(__file__)
 
-@pytest.fixture
 def capability():
      return capnp.load(os.path.join(this_dir, 'test_capability.capnp'))
 
+@pytest.fixture(name="capability")
```
Comment (cjermain) [inline test/test_capability_context.py]: Can you explain why you are using a second wrapper for the fixture?

File: test/test_rpc_calculator.py [hunk 2]
```diff
@@ -20,9 +20,9 @@ def test_calculator():
 
 
 def run_subprocesses(address):
-    server = subprocess.Popen([examples_dir + '/calculator_server.py', address])
+    server = subprocess.Popen([sys.executable, examples_dir + '/calculator_server.py', address])
```
Comment (cjermain) [inline test/test_rpc_calculator.py]: This change looks good.

**Comment by haata** [comment]
Thanks for this PR (I used it for some of my test fixes, though I didn't apply it directly).

**Pull request closed without merging**